src/main/java/com/royalrangers/model/VerificationToken.java

```diff
@@ -4,49 +4,47 @@
 import lombok.Setter;
 
 import javax.persistence.*;
-import java.sql.Timestamp;
-import java.util.Calendar;
 import java.util.Date;
 
+import static com.royalrangers.service.TokenService.calculateTokenExpiryDate;
+
 @Getter
 @Setter
 @Entity
 public class VerificationToken extends BaseModel {
 
     private static final int EXPIRATION = 60 * 24;
 
-    private String token;
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
```

Comment:
Why did you add explicitly 'id' into this class? We have already had 'id' into 'BaseModel'.